Branch-SDK/Branch-SDK/Branch.m

```diff
@@ -521,49 +491,33 @@ - (void)setRequestMetadataKey:(NSString *)key value:(NSObject *)value {
 }
 
 - (void)enableDelayedInit {
-    @synchronized (self.shouldWaitForInitLock) {
-        self.preferenceHelper.shouldWaitForInit = YES;
-    }
-
-    self.useCookieBasedMatching = NO; // Developers delaying init should implement their own SFSafariViewController
+    // deprecate this
 }
 
 - (void)disableDelayedInit {
-    @synchronized (self.shouldWaitForInitLock) {
-        self.preferenceHelper.shouldWaitForInit = NO;
-    }
+    // deprecate this
 }
 
 - (NSURL *)getUrlForOnboardingWithRedirectUrl:(NSString *)redirectUrl {
-    return [BNCStrongMatchHelper getUrlForCookieBasedMatchingWithBranchKey:self.class.branchKey redirectUrl:redirectUrl];
+    // deprecate this
+    return nil;
 }
 
 - (void)resumeInit {
-    @synchronized (self.shouldWaitForInitLock) {
-        self.preferenceHelper.shouldWaitForInit = NO;
-    }
-    if (self.initializationStatus == BNCInitStatusInitialized) {
-        BNCLogError(@"User session has already been initialized, so resumeInit is aborting.");
-    }
-    else if (![self.requestQueue containsInstallOrOpen]) {
-        BNCLogError(@"No install or open request, so resumeInit is aborting.");
-    }
-    else {
-        [self processNextQueueItem];
-    }
+    // deprecate this
 }
 
 - (void)setInstallRequestDelay:(NSInteger)installRequestDelay {
-    self.preferenceHelper.installRequestDelay = installRequestDelay;
+    // deprecate this
```

Comment:
Will utilizing these a good idea?: 
https://stackoverflow.com/questions/3908715/how-do-i-flag-a-method-as-deprecated-in-objective-c-2-0

Reply:
Yea i forgot to add the deprecation warnings.